test/verify/check-realms

```diff
@@ -197,11 +197,25 @@ if ! grep -q 'services.*nss' /etc/sssd/sssd.conf; then
     systemctl restart sssd
 fi
 
-# This needs to work
-getent passwd admin@cockpit.lan
+# HACK: This needs to work, but may take a minute
+for x in $(seq 1 60); do
```

Comment:
Yup. It used to be factorial 20!, which helped but didn't solve the issue ... after bumping to 60! it ran for hours straight without failure.